src/xunit.performance.api/XunitBenchmark.cs

```diff
@@ -52,6 +54,15 @@ static XunitBenchmark()
                     ),
                 }
             };
+
+            foreach (var pmc in PerformanceMonitorCounters)
```

Comment:
Yes, we are forcing it on for the next perf review meeting to capture statistics that can help correlate data and benchmark noise.